src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
+import { PostInfo } from '../PostInfo';
+import { PostArr } from '../../type/post';
+import './PostList.scss';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+export const PostList: React.FC<PostArr> = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo post={post} />
```

Comment:
Add a 'key' prop to the 'PostInfo' component with a unique identifier, such as 'post.id', to improve performance.